scripts/prerm

```diff
@@ -1,10 +1,6 @@
 #!/bin/sh
 
 # Cleanup symbolic links created during postinst
-rm -f ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages/TestHarness
-rm -f ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/bin
-
-# Attempt to clean up directories that may have been created during postinst
-# Also may have already existed, so it is acceptable to leave them in place if the rmdir fails
-rmdir --ignore-fail-on-non-empty ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages
-rmdir --ignore-fail-on-non-empty ${CMAKE_INSTALL_FULL_LIBDIR}/python3
+Python_DISTLIB=$(python3 -c "import sysconfig; print(sysconfig.get_path('purelib'))" | sed 's/site-packages/dist-packages/')
```

Comment:
I really don't think this is the right answer.

Reply:
I agree. Something feels wrong about the trouble we're having here and that's usually a reasonable indicator we're on the wrong path.

Reply:
This implementation is based on the description of https://wiki.debian.org/Python, first point of the section `Deviations from upstream`. In debian/Ubuntu, using the deprecated `distutils.sysconfig.get_python_lib(plat_specific=False,standard_lib=False)` does return the right answer `/usr/lib/python3/dist-packages` while using sysconfig does not. Based on the [StackOverflow post](https://stackoverflow.com/questions/122327/how-do-i-find-the-location-of-my-python-site-packages-directory) 

> The root cause of the discrepancy is because Debian [patch the distutils install layout](https://salsa.debian.org/cpython-team/python3/-/blob/master/debian/patches/distutils-install-layout.diff), to correctly reflect their changes to the site, but they fail to patch the sysconfig module.

I am not sure how to move forward from here. Here are the list of solutions I can think of (I am welcome any other solutions)
- just use `/usr/lib/python3/dist-packages`,
- use distutils (at least it's still available in our supported distributions),
- keep the current implementation.

Reply:
I think my vote is to just use distutils and sort out the problem with python 3.12 + ubuntu when that actually arrives.

Reply:
As much as I dislike using distutils, it seems like the least painful solution.